packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -149,35 +153,35 @@ export class ProfilesUtils {
-     * scopes since order presedence can be hard to predict based on the user's setup
+     * scopes since order precedence can be hard to predict based on the user's setup
      */
-    public static async promptAndDisableCredentialManagement(): Promise<void> {
-        ZoweLogger.trace("ProfilesUtils.promptAndDisableCredentialManagement called.");
-        const noButton = vscode.l10n.t("No");
-        const yesGloballyButton = vscode.l10n.t("Yes, globally");
-        const yesWorkspaceButton = vscode.l10n.t("Only for this workspace");
-        const response = await Gui.warningMessage(
-            vscode.l10n.t("Zowe Explorer failed to activate since the default credential manager is not supported in your environment."),
-            {
-                items: [noButton, yesGloballyButton, yesWorkspaceButton],
-                vsCodeOpts: {
-                    modal: true,
-                    detail: vscode.l10n.t("Do you wish to disable credential management? (VS Code window reload will be triggered)"),
-                },
-            }
-        );
-        if (response === yesGloballyButton || response === yesWorkspaceButton) {
-            const scope = response === yesGloballyButton ? vscode.ConfigurationTarget.Global : vscode.ConfigurationTarget.Workspace;
-            await SettingsConfig.setDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, false, scope);
-            await vscode.commands.executeCommand("workbench.action.reloadWindow");
-        } else {
-            throw new imperative.ImperativeError({
-                msg: vscode.l10n.t(
+    public static async disableCredentialManagement(): Promise<void> {
+        ZoweLogger.trace("ProfilesUtils.disableCredentialManagement called.");
+        const settingEnabled: boolean = SettingsConfig.getDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, true);
+        if (settingEnabled) {
+            this.PROFILE_SECURITY = false;
+            await SettingsConfig.setDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, false, vscode.ConfigurationTarget.Global);
+            await Gui.infoMessage(
+                vscode.l10n.t(
                     // eslint-disable-next-line max-len
-                    "Failed to load credential manager. This may be related to Zowe Explorer being unable to use the default credential manager in a browser based environment."
+                    "Zowe Explorer's default credentials manager is not supported in your environment. Consider installing a custom credential manager for your platform. Click Reload to start Zowe Explorer without a credentials manager."
                 ),
-            });
+                {
+                    items: [vscode.l10n.t("Reload window")],
+                }
+            );
+            await vscode.commands.executeCommand("workbench.action.reloadWindow");
+        }
+    }
+
+    public static checkDefaultCredentialManager(): boolean {
```

Comment:
Ok, will push a commit but that async change had quite a ripple effect. I kept the method to get a special Zowe Explorer log message.

Reply:
The test for the credentials manager needs to be done before a ProfileInfo object can be created. So it would be a chicken-egg problem to solve. `ProfilesCache.requireKeyring();` seems to already be a valid abstraction for the test as well.